Sitemap/Voter.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace Symfony\Cmf\Bundle\SeoBundle\Sitemap;
+
+/**
+ * A chain as a decorator for all tagged voter services.
+ *
+ * @author Maximilian Berghoff <Maximilian.Berghoff@mayflower.de>
+ */
+class Voter extends AbstractChain implements VoterInterface
```

Comment:
the publish workflow in core bundle is doing this successfully, could be an example to look at how to do this.